samples/msal-react-samples/gatsby-sample/jest.config.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
     displayName: "Gatsby",
     globals: {
         __PORT__: 3001,
-        __STARTCMD__: "npm run serve -- -p 3001"
+        __STARTCMD__: "NODE_ENV=development npm run develop -- -p 3001"
```

Comment:
Why the change here from serving the production version to development?

Reply:
the build step is happening before the test files run to replace the configuration parameters. So if we run serve here we won't be not running the current state of the project files. I didn't realize it matters though (isn't this what we do with other react samples [e.g.](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/f3f54cbc2faf30894852a98585e22c5869277858/samples/msal-react-samples/typescript-sample/jest.config.js#L5)?